app/controllers/rescue_requests_controller.rb

```diff
@@ -15,12 +15,12 @@ def index
 
   def disaster_index
     status_ids = [RequestStatus.find_by(name: 'Rescued').id, RequestStatus.find_by(name: 'Closed').id]
-    status_query = status_ids.map { |s| "request_status_id = #{s}" }.join(' OR ')
+    status_query = (status_ids.map { |s| "request_status_id = #{s}" } + ["(dupe_of > 0 AND dupe_of is not null)"]).join(' OR ')
```

Comment:
To simplify this, we should set a request's status to Closed when it's marked as a duplicate, so that we don't have to have the extra clause in this query.

Also, SQL goes in SHOUTY_CASE - `dupe_of IS NOT NULL`.

Reply:
I don't want to mark them closed becuase their previous status could be important and we haven't logged normal triage status chamges yet.

Reply:
...so let's log triage status changes?

Reply:
Done